### ISSUE TITLE
[engine] Flutter fuchsia tests are failing

### ISSUE THREAD

**Issue by godofredoc** [flutter/flutter#45222]
Fuchsia tests are failing with the following message:
```
[ERROR:flutter/shell/platform/fuchsia/flutter/accessibility_bridge.cc(24)] Flutter cannot connect to SemanticsManager with status: ZX_ERR_CANCELED.
[ERROR:flutter/shell/platform/fuchsia/flutter/accessibility_bridge.cc(24)] Flutter cannot connect to SemanticsManager with status: ZX_ERR_CANCELED.
[ERROR:flutter/shell/platform/fuchsia/flutter/accessibility_bridge.cc(24)] Flutter cannot connect to SemanticsManager with status: ZX_ERR_CANCELED.
[ERROR:flutter/shell/platform/fuchsia/flutter/accessibility_bridge.cc(24)] Flutter cannot connect to SemanticsManager with status: ZX_ERR_CANCELED.
[ERROR:flutter/shell/platform/fuchsia/flutter/accessibility_bridge.cc(110)] Semantics Node 0 has already been listed as a child of another node, ignoring for parent 0.
[ERROR:flutter/shell/platform/fuchsia/flutter/accessibility_bridge.cc(110)] Semantics Node 0 has already been listed as a child of another node, ignoring for parent 0.
[ERROR:flutter/shell/platform/fuchsia/flutter/accessibility_bridge.cc(110)] Semantics Node 0 has already been listed as a child of another node, ignoring for parent 1.
[ERROR:flutter/shell/platform/fuchsia/flutter/accessibility_bridge.cc(24)] Flutter cannot connect to SemanticsManager with status: ZX_ERR_CANCELED.
```
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)

Link to tasks:
https://chromium-swarm.appspot.com/task?id=489f4c2f572b0310
https://chromium-swarm.appspot.com/task?id=489f4c0135ef5a10

**Comment by iskakaushik** [comment]
These are fixed now.
